.gitignore

```diff
@@ -17,6 +17,9 @@ kustomization.yaml
 # MongoDB global configs
 globalConfig.json
 
+# Mapbox secrets.
+boundaries.json
```

Comment:
Should we all have this locally?

Reply:
plan is to make it optional for dev, but if you have it you can use it yes (most likely via an environment var pointing to it, not sure yet how I'm going to plug it)